test/errors/useImportUnknownSym-detailed.good

```diff
@@ -0,0 +1,11 @@
+─── error in useImportUnknownSym.chpl:6 [UseImportUnknownSym] ───
+  Cannot find symbol 'b' for import.
+  Symbol named here:
```

Comment:
Personally, I'd once again go with "In the following {use,import} statement", or with a mix of both, "The symbol was referenced in the following {use,import} statement:"